web/src/lib/models/heplers.ts

```diff
@@ -0,0 +1,21 @@
+import { EmbeddingModelDescriptor, FullEmbeddingModelDescriptor, INVALID_OLD_MODEL } from "@/interfaces/admin/models/models";
+
+export function checkModelNameIsValid(modelName: string | undefined | null) {
```

Comment:
lets use ternary, that way it will be much shorter and readable, like this:

!modelName || modelName === INVALID_OLD_MODEL ? false : true